frontends/queues/queues/strict_or_rr.py

```diff
@@ -125,7 +125,7 @@ def insert_queue(
         for b in range(numflows)
     ]
     invoke_subqueues_value_guard = cb.par(
-        invoke_subqueues_value_guard_seq
+        *invoke_subqueues_value_guard_seq
```

Comment:
Interesting, so how was this working out before?

Reply:
This is just a silly python thing.

If you throw a list into a function that takes a variable number of args, it's not that the function is called with list entries as arguments. Instead, the function is called with one argument: the entire list.

For example, try running this:
```
def f(*args):
    for arg in args:
        print(arg)

print("calling f([1, 2, 3])")
f([1,2,3])
print("calling f(*[1, 2, 3])")
f(*[1,2,3])
```

For our code, this means we were throwing a list (i.e. a `seq` block) into `cb.par`. So, the resulting `futil` code would look like 
```
par { 
    seq {
        ... # guarded subqueue invokes
    }
} 
```
instead of 
```
par { 
     ... # guarded subqueue invokes
} 
```
Both are fine for correctness, but I think we're accidentally not running them in parallel.

Reply:
Ah gotcha, I think I have caught something a little like this before but I needed my memory jogged! Probably a good thing for us to keep in mind as we try to fine-tune our implementations. This sort of thing is money left on the table!